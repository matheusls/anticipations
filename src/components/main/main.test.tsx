import { render, screen } from '@testing-library/react';

import Main from './main';

const children = 'Render of Main component';

describe('<Main />', () => {
  it('should match snapshot', () => {
    const { container } = render(<Main>{children}</Main>);

    expect(container).toMatchSnapshot();
  });

  it('should render Main correctly', () => {
    render(<Main>{children}</Main>);

    expect(screen.getByRole('main')).toHaveTextContent(children);
  });
});
