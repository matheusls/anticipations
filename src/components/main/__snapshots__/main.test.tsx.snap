// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main /> should match snapshot 1`] = `
<div>
  <main
    class=""
  >
    Render of Main component
  </main>
</div>
`;
